build.gradle

```diff
@@ -230,6 +230,8 @@ project(':iceberg-api') {
   dependencies {
     implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
     compileOnly "com.google.errorprone:error_prone_annotations"
+    annotationProcessor "org.immutables:value"
+    implementation "org.immutables:value"
```

Comment:
I thought you had said that this didn't require any additional runtime dependencies? We need to go through and validate the license and license documentation (NOTICE).

Reply:
technically it's a `compileOnly` dependency, I just forgot to set it properly to `compileOnly`